cobigen-templates/pom.xml

```diff
@@ -31,6 +31,7 @@
     <resources>
       <resource>
         <directory>${basedir}</directory>
+        <directory>src/main/templates</directory>
         <excludes>
```

Comment:
To have a valid configuration now, you should remove basedir from the resource configurations to not double the sources in the jar and remove the exclusions as they are not needed anymore. Please check the jar build by mvn clean install (in the target folder)